src/components/pipelines-overview/PipelinesOverviewPage.tsx

```diff
@@ -100,5 +120,6 @@ const PipelinesOverviewPage: React.FC = () => {
     </>
   );
 };
+import { IntervalOptions, TimeRangeOptions, useQueryParams } from './utils';
```

Comment:
nit: move it to the top with all imports.

Reply:
yep cool will do.